Gemfile

```diff
@@ -141,3 +141,5 @@ gem 'active_model_serializers', github: 'rails-api/active_model_serializers', re
 gem 'aws-sdk', '~> 2'
 
 gem 'rubocop', require: false, group: [:development, :staging]
+# Reduce volume of production logs
+gem 'lograge'
```

Comment:
I tried `gem 'lograge', group: :production`, ran `bundle install --without staging,test,levelbuilder`, `RAILS_ENV=production bundle exec rails s`, and received the error:

`